### DISCUSSION TITLE
Change speed for G28 and G30 button

### DISCUSSION THREAD

**Discussion by rmaltais** [terjeio/ioSender#329] [General]
Hi,

We have a button for go to G28 and G30. This button always travel with maximum speed. Can i change speed in setting for this button witout change my maximum speed travel?

Thank

**Comment by terjeio** [comment]
> Can i change speed in setting for this button witout change my maximum speed travel?

No, [it is not possible](http://linuxcnc.org/docs/html/gcode/g-code.html#gcode:g28-g28.1).
